doc/user/input-data.rst

```diff
@@ -91,8 +98,7 @@ Metadata files below describe input data and are required by `qserv-ingest`:
     }
 
 - `<database_name>.json`: describe the database to register inside the replication service and where the data will be ingested,
-  its format is described in `replication service documentation <https://confluence.lsstcorp.org/pages/viewpage.action?pageId=133333850#UserguidefortheQservIngestsystem(APIversion1)-RegisteringanewdatabaseinQserv>`__.
 - `<table_name>.json`: each of these files describes a table to register inside the replication service and where the data will be ingested,
-  its format is described in `replication service documentation <https://confluence.lsstcorp.org/pages/viewpage.action?pageId=133333850#UserguidefortheQservIngestsystem(APIversion1)-Registeringatable>`__.
 - `<table_index>.json`:each of these files describes an index to create for a given set of chunk tables,
-  its format is described in `replication service documentation <https://confluence.lsstcorp.org/display/DM/Managing+indexes#Managingindexes-Request>`__.
+
+Formats for all of these files are documented in `replication service documentation <https://confluence.lsstcorp.org/pages/viewpage.action?pageId=133333850>`__.
```

Comment:
Why are there double underscores after the link? I thought that RST needs just one underscore?

Reply:
:+1: